.github/workflows/update_version_numbers.yml

```diff
@@ -38,14 +38,19 @@ jobs:
           sed -i '' "s#detail.version_minor =.*#detail.version_minor = $package_version_minor;#" ./src/tests/class_tests/openms/source/VersionInfo_test.cpp
           sed -i '' "s#detail.version_patch =.*#detail.version_patch = $package_version_patch;#" ./src/tests/class_tests/openms/source/VersionInfo_test.cpp
 
+          # update vcpkg.json
+          sed -i "s/\"version-string\": \".*\"/\"version-string\": \"$package_version\"/" vcpkg.json
+
           # update test write ini out:
           sed -i '' "s#<ITEM name=\"version\" value=\".*\" type=\"string\"#<ITEM name=\"version\" value=\"$package_version\" type=\"string\"#g" ./src/tests/topp/WRITE_INI_OUT.ini
           # update INIUpdater version
           sed -i '' "s#<ITEM name=\"version\" value=\".*\" type=\"string\"#<ITEM name=\"version\" value=\"$package_version\" type=\"string\"#g" ./src/tests/topp/INIUpdater_1_noupdate.toppas
 
 
           # update INIs in tests topp:
-          find ./src/tests/topp/ -type f -name '*.ini' -exec grep -q "<ITEM name=\"version\" value=\".*\" type=\"string\"" {} \; -exec sed -i '' "s#<ITEM name=\"version\" value=\".*\" type=\"string\"#<ITEM name=\"version\" value=\"$package_version\" type=\"string\"#g" {} \;
+          find $TOOL_DIR_PATH -type f -iname '*.ini' \
+            -exec grep -q '<ITEM name="version".*Version of the tool' {} \; \
```

Comment:
is this the proper syntax ? *Version of the tool

Reply:
yes, but staring at this again, I don't think it needs the changes that I added. I'll roll them back